components/Admin/Embed/EmbedTodayPrayerTimes/EmbedTodayPrayerTimes.tsx

```
@@ -0,0 +1,194 @@
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select"
+import { CheckIcon } from "lucide-react"
+import { cn } from "@/lib/utils"
+
+export default function EmbedTodayPrayerTimes({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)